tests/simulate.py

```diff
@@ -0,0 +1,119 @@
+#!/usr/bin/env python3
+
+"""
+Create a simulated Measurement Set for use in comparing a number of imagers
+against each other.
+
+The time range, frequencies and phase centre are hard-coded in the script,
+while the local sky model (consisting of point sources with polarisation but
+no spectral shape) is read from lsm.txt, and the antennas from
+meerkat_antennas.txt.
+"""
+
+import os
+import sys
+import unittest.mock
+
+import numpy as np
+import pandas
+import katpoint
+import casacore.tables
+
+import astropy.units as u
+from astropy.coordinates import SkyCoord, Angle
+from astropy.time import Time
+import astropy.io.ascii
+
+# RASCIL 0.2.0b0 imports nifty_gridder but doesn't install it. It's not needed
+# for this simulation, so just mock it out.
+sys.modules['nifty_gridder'] = unittest.mock.Mock()
+# RASCIL gets upset if it can't find its data directory, but doesn't actually
+# need it for this simulation. Just give it a fake one.
+if 'RASCIL_DATA' not in os.environ:
+    os.environ['RASCIL_DATA'] = '/'
+
+import rascil.data_models                   # noqa: E402
+import rascil.processing_components         # noqa: E402
+
+
+freqs = 856e6 + 214e6 * (np.arange(4) + 0.5)
+start_time = Time('2014-01-01T15:00:00.0', format='isot', scale='utc')
+time_step = 4.0 * u.s
+times = start_time + time_step * (np.arange(3600) + 0.5)
+phase_centre = SkyCoord(ra='3h30m00s', dec='-35d00m00s', frame='icrs')
+
+lsm = astropy.io.ascii.read('lsm.txt')
```

Comment:
Using a `katpoint.Catalogue` to define the sky model gives you the freedom of adding spectral shapes to the sources in case you want to do that in future. Also easily constructed from a text file.

Reply:
I thought about it, but this approach is flexible enough to allow things like Gaussian sources that can't be specified in katpoint. Spectral shape is not that useful because for a spectral image I only care about the flux at one frequency.